implementations/rust/daemon/src/cli.rs

```diff
@@ -37,15 +37,21 @@ pub struct Args {
         default_value = "stdout",
         help = r#"Route to channel responder, e.g. udp://host:port[,udp://host:port] (note comma-separation) or "stdout""#
     )]
-    route: OutputKind,
+    sink_route: OutputKind,
```

Comment:
this breaks the CLI -- i don't think we need to disambiguate the `sink_route` to end-users, as (I believe) there's only 1 kind of `route`.

Reply:
ok